lib/config/presets/internal/packages.ts

```diff
@@ -36,18 +36,28 @@ export const presets: Record<string, Preset> = {
     description: 'All Stylelint packages.',
     matchPackagePrefixes: ['stylelint'],
   },
+  standard: {
+    description: 'JavaScript Standard Style',
+    matchPackageNames: ['standard'],
+  },
   tslint: {
     description: 'All TSLint packages.',
     matchPackageNames: ['codelyzer'],
     matchPackagePatterns: ['\\btslint\\b'],
   },
+  prettier: {
+    description: 'Prettier code formatter',
+    matchPackageNames: ['prettier'],
+  },
   linters: {
     description: 'All lint-related packages.',
     extends: [
       'packages:emberTemplateLint',
       'packages:eslint',
       'packages:stylelint',
       'packages:tslint',
+      'packages:prettier',
+      'packages:standard',
     ],
     matchPackageNames: ['remark-lint'],
```

Comment:
can't we simply add those two packages to this matcher?

Reply:
To line 62?

Reply:
Yes, good idea